src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -52,7 +52,7 @@ export interface ITerminalInternalOptions {
 	isFeatureTerminal?: boolean;
 	useShellEnvironment?: boolean;
 	isSplitTerminal?: boolean;
-	target?: TerminalLocation;
+	parentTerminal?: ExtHostTerminal;
```

Comment:
Should this and `isSplitTerminal` be similar to `location: TerminalLocation | vscode.TerminalEditorLocationOptions | vscode.TerminalSplitLocationOptions`?